env-config/config.py

```diff
@@ -74,14 +74,14 @@
 NGINX_PORT = '443'
 BASE_URL = 'https://{}/'.format(FQDN)
 
-SECRET_KEY = '<INSERT_RANDOM_STRING_HERE>'
+SECRET_KEY = 'fezfjepiozjpozfejpozfejkpzfejkpzfepzfej'
```

Comment:
You shouldn't have committed this file.

Reply:
True, my bad, in fact this is just dev value so not critical at all, will repushed proper config.py